packages/common/src/debounce.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Creates a debounced function that waits "ms" milliseconds before being called.
```

Comment:
Creates a debounced function that waits `ms` milliseconds between invocations. If the function is called multiple times between invocations, the latest arguments passed to the function will be used.